### PR TITLE
Converted LISTVOLUME command in gRPC

### PR DESCRIPTION
**Pull request by vishal-shakya-ssir**
This PR has implementation of LISTVOLUME command into gRPC

### CODE REVIEW

File: src/event/pos_event.yaml [hunk 1]
```diff
@@ -1228,6 +1228,14 @@ Root:
     Cause: State of the input array is not online.
     Solution:
 
+  -
+    Id: 1878
+    Name: LIST_VOL_ARRAY_NAME_DOES_NOT_EXIST
```
Comment (mjlee34) [inline src/event/pos_event.yaml:1233]: LIST_VOLUME_ ... can be better for consistency with others?

**Comment by mjlee34** [comment]
Just one thing to do. As LISTARRAY command displays the response in human-readable form, display_responses.go needs to be changed accordingly. 👍

File: src/event/pos_event.yaml (hunk 1 above)
Comment (vishal-shakya-ssir) [reply src/event/pos_event.yaml:1233]: As discussed, This is consistent with other events name 
Thanks

**Comment by vishal-shakya-ssir** [comment]
Thanks for pointing out,
I will change  display_responses.go

**Review by mjlee34 — DISMISSED** [review]

**Review by hsung-yang — APPROVED** [review]

**Review by junghyun-34 — DISMISSED** [review]

**Review by ringopow — APPROVED** [review]
수고하셨습니다

**Review by aswinssir — APPROVED** [review]
LGTM

**Review by mjlee34 — APPROVED** [review]

**Review by junghyun-34 — APPROVED** [review]

**Pull request merged**